server/src/weaverbird/pipeline/steps/date_extract.py

```diff
@@ -45,6 +49,20 @@
     ],
 ]
 
+DATE_UNIT_WITH_INT_CAST_NOT_NEEDED: list[str] = [
```

Comment:
Seems like a duplicate of `FIRST_DAY_DATE_PARTS`, is there a reason to have both ? Also, why is this all lowercase ?

Reply:
One is a list, the other one is a type declaration